src/Microsoft.Azure.WebJobs.Host/FunctionExecutingContext.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using Microsoft.Azure.WebJobs.Host.Executors;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Host
+{
+    /// <summary>
+    /// The context for an executed function
+    /// </summary>
+    [CLSCompliant(false)]
```

Comment:
This assembly is CLS compliant, but these types are using non CLS compliant constructs, or at least they were. Hamza, if you remove these, what does the compiler say the issue is? I can't remember. Maybe they're not needed anymore.